### PR TITLE
CopyOfH1

### PR DESCRIPTION
**Pull request by Jukis252**
Created pull request of copy H1, since I have already fixed the code second time a month ago :)

### CODE REVIEW

File: CalculateBetweenBiggestElementAndSmallestElement.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    public class CalculateBetweenBiggestElementAndSmallestElement
+    {
+        public void SumOfBiggestElementAndSmallest()
+        {
+            List<int> array = new List<int>();
+            Console.WriteLine("Enter the how much numbers will you input: ");
+            int imputOfNumber = int.Parse(Console.ReadLine());
+            for (int index = 0; index < imputOfNumber; index++)
+            {
+                Console.WriteLine("Input "+(index+1)+" number: ");
```
Comment (radislavkireev) [inline CalculateBetweenBiggestElementAndSmallestElement.cs]: add spaces for readability 
`
 Console.WriteLine("Input "+ (index + 1) + " number: ");`

File: CalculateBetweenBiggestElementAndSmallestElement.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        int SumBetweenBiggestAndSmallestElementsInArray(List<int> array, int biggestPosition, int smallestPosition)
+        {
+            int Sum=0;
+            if(biggestPosition > smallestPosition)
+            {
+                for(int index = smallestPosition; index <= biggestPosition;index++)
+                {
+                    Sum=Sum+array[index];
```
Comment (radislavkireev) [inline CalculateBetweenBiggestElementAndSmallestElement.cs]: This place and others with plus symbol - add spaces for readability.
`
 Sum = Sum + array[index];`

File: DecimalToTenaryConvertion.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            }
+        }
+
+        string ConvertDecimalToTernary(int number)
+        {
+            string tenaryNumber = "";
+            while(number > 0)
+            {
+                int remainder = number % 3;
+                remainder.ToString();
```
Comment (radislavkireev) [inline DecimalToTenaryConvertion.cs]: drop this line

File: FindLastDigitOfISBN.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            Console.WriteLine("Input a 9 numbers in one line: ");
+            string inputNumber =  Console.ReadLine();
+            char[] arrayOfChars = inputNumber.ToCharArray();
+            int ISBN = 0;
+            for (int index = 0; index < arrayOfChars.Length; index++)
+            {
+                ISBN = ISBN + (10-index)*(arrayOfChars[index]-48);
+            }
+            int lastDigit = FindDigit(ISBN);
+            {
```
Comment (radislavkireev) [inline FindLastDigitOfISBN.cs]: drop {

File: FindLastDigitOfISBN.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                Console.WriteLine(inputNumber+lastDigit);
+            }
```
Comment (radislavkireev) [inline FindLastDigitOfISBN.cs]: drop }

**Review by radislavkireev — APPROVED** [review]

**Review by Jukis252 — COMMENTED** [review]
Fixed all mistakes

**Pull request merged**